lib/election.rb

```diff
@@ -1,2 +1,27 @@
 class Election
+  attr_reader :year, :races, :candidates
+
+  def initialize(year)
+    @year = year
+    @races = []
+    @candidates = []
+  end
+
+  def add_race(office)
+    @races << Race.new(office)
+    return @races
+  end
+
+  def add_candidate_to_election(candidate_info)
```

Comment:
This method appears to be doing almost the same thing as register candidate - do you need both?